rust/operator-binary/Cargo.toml

```diff
@@ -10,7 +10,7 @@ version = "0.12.0-nightly"
 publish = false
 
 [dependencies]
-stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "0.27.1" }
+stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "0.35.0" }
```

Comment:
**nit:** `clap` and `tokio` could be set to the same versions as in operator-rs, i.e.

```toml
clap = "4.1"
tokio = { version = "1.25", features = ["full"] }
```

Reply:
done https://github.com/stackabletech/opa-operator/pull/411/commits/0acc8fcc3e4fcaaba3c663739fc8162be5c084c1